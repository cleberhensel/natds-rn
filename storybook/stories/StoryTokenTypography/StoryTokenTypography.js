import React from 'react';
import { ScrollView, Dimensions, SafeAreaView } from 'react-native';
import { NatContainer, NatText, ThemeNaturaLight } from '../../../lib';

const styles = {
  defaultScreen: {
    flex: 1,
    width: Dimensions.get('window').width,
  },
};

export const StoryTokenTypography = () => (
  <SafeAreaView style={styles.defaultScreen}>
    <NatContainer>
      <ScrollView>
        <NatText value="SIZES" type="h6" />
        <NatText value="h1. Heading" type="h1" />
        <NatText value="h2. Heading" type="h2" />
        <NatText value="h3. Heading" type="h3" />
        <NatText value="h4. Heading" type="h4" />
        <NatText value="h5. Heading" type="h5" />
        <NatText value="h6. Heading" type="h6" />
        <NatText value="subtitle1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur" type="subtitle1" />
        <NatText value="subtitle2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur" type="subtitle2" />
        <NatText value="body1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam" type="body1" />
        <NatText value="body2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos" type="body2" />
        <NatText value="BUTTON TEXT" type="button" />
        <NatText value="Caption text" type="caption" />
        <NatText value="OVERLINE TEXT" type="overline" />

        <NatText value="VARIATIONS" type="h6" />
        <NatText value="body1. Set diffente color" type="body1" color={ThemeNaturaLight.palette.primary.main} />
        <NatText value="body1. Withou marginBottom spacingNone" type="body1" margin={ThemeNaturaLight.spacing.spacingNone} />
        <NatText value="body1. Withou marginBottom spacingHuge" type="body1" margin={ThemeNaturaLight.spacing.spacingHuge} />
        <NatText value="body1. Withou marginBottom spacingMicro" type="body1" margin={ThemeNaturaLight.spacing.spacingMicro} />
      </ScrollView>
    </NatContainer>
  </SafeAreaView>

);
