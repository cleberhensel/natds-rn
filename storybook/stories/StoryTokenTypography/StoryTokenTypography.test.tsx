import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components/native";
import { StoryTokenTypography } from "./StoryTokenTypography";
describe("StoryTokenTypography component", () => {
  it("Should render StoryNatCard component ", () => {
    const button = renderer.create(<StoryTokenTypography />).toJSON();
    expect(button).toMatchSnapshot("StoryTokenTypography snapshot");
  });
});
