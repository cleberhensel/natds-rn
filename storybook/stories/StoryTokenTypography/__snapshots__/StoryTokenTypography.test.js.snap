// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryNatCard component Should render StoryNatCard component  1`] = `
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Array [
        Object {
<<<<<<< HEAD
          "backgroundColor": "#FFFFFF",
=======
          "backgroundColor": "white",
>>>>>>> origin/update/react
          "bottom": 0,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "left": 0,
          "paddingBottom": 0,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 0,
          "right": 0,
          "top": 0,
        },
        Object {
          "backgroundColor": "#FFF",
<<<<<<< HEAD
          "marginTop": 0,
=======
<<<<<<< HEAD
          "marginTop": 0,
=======
<<<<<<< HEAD
          "marginTop": 0,
=======
          "marginTop": 30,
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
          "width": 750,
        },
      ]
    }
  >
    <RCTScrollView>
      <View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 20,
                  "fontWeight": "500",
                  "lineHeight": 30,
                },
              ]
            }
            type="h6"
          >
            SIZES
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 96,
                  "fontWeight": "400",
                  "lineHeight": 144,
                },
              ]
            }
            type="h1"
          >
            h1. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 60,
                  "fontWeight": "400",
                  "lineHeight": 90,
                },
              ]
            }
            type="h2"
          >
            h2. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 48,
                  "fontWeight": "400",
                  "lineHeight": 72,
                },
              ]
            }
            type="h3"
          >
            h3. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 34,
                  "fontWeight": "400",
                  "lineHeight": 51,
                },
              ]
            }
            type="h4"
          >
            h4. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 24,
                  "fontWeight": "400",
                  "lineHeight": 36,
                },
              ]
            }
            type="h5"
          >
            h5. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 20,
                  "fontWeight": "500",
                  "lineHeight": 30,
                },
              ]
            }
            type="h6"
          >
            h6. Heading
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "500",
                  "lineHeight": 24,
                },
              ]
            }
            type="subtitle1"
          >
            subtitle1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 14,
                  "fontWeight": "500",
                  "lineHeight": 21,
                },
              ]
            }
            type="subtitle2"
          >
            subtitle2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            body1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 14,
                  "fontWeight": "400",
                  "lineHeight": 21,
                },
              ]
            }
            type="body2"
          >
            body2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 14,
                  "fontWeight": "500",
                  "lineHeight": 21,
                },
              ]
            }
            type="button"
          >
            BUTTON TEXT
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 12,
                  "fontWeight": "400",
                  "lineHeight": 18,
                },
              ]
            }
            type="caption"
          >
            Caption text
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 12,
                  "fontWeight": "500",
                  "lineHeight": 18,
                },
              ]
            }
            type="overline"
          >
            OVERLINE TEXT
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 20,
                  "fontWeight": "500",
                  "lineHeight": 30,
                },
              ]
            }
            type="h6"
          >
            VARIATIONS
          </Text>
        </View>
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#F4AB34"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#F4AB34",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            body1. Set diffente color
          </Text>
        </View>
        <View
          margin={0}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            body1. Withou marginBottom spacingNone
          </Text>
        </View>
        <View
          margin={64}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 64,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            body1. Withou marginBottom spacingHuge
          </Text>
        </View>
        <View
          margin={4}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            body1. Withou marginBottom spacingMicro
          </Text>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
=======
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "bottom": 0,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "left": 0,
        "paddingBottom": 0,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 0,
        "right": 0,
        "top": 0,
      },
      Object {
        "backgroundColor": "#FFF",
        "marginTop": 30,
        "width": 750,
      },
    ]
  }
>
  <RCTScrollView>
    <View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 20,
                "fontWeight": "500",
                "lineHeight": 30,
              },
            ]
          }
          type="h6"
        >
          SIZES
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 96,
                "fontWeight": "400",
                "lineHeight": 144,
              },
            ]
          }
          type="h1"
        >
          h1. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 60,
                "fontWeight": "400",
                "lineHeight": 90,
              },
            ]
          }
          type="h2"
        >
          h2. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 48,
                "fontWeight": "400",
                "lineHeight": 72,
              },
            ]
          }
          type="h3"
        >
          h3. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 34,
                "fontWeight": "400",
                "lineHeight": 51,
              },
            ]
          }
          type="h4"
        >
          h4. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 24,
                "fontWeight": "400",
                "lineHeight": 36,
              },
            ]
          }
          type="h5"
        >
          h5. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 20,
                "fontWeight": "500",
                "lineHeight": 30,
              },
            ]
          }
          type="h6"
        >
          h6. Heading
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 16,
                "fontWeight": "500",
                "lineHeight": 24,
              },
            ]
          }
          type="subtitle1"
        >
          subtitle1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 14,
                "fontWeight": "500",
                "lineHeight": 21,
              },
            ]
          }
          type="subtitle2"
        >
          subtitle2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
            ]
          }
          type="body1"
        >
          body1. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 14,
                "fontWeight": "400",
                "lineHeight": 21,
              },
            ]
          }
          type="body2"
        >
          body2. Lorem ipsum dolor sit amet, consectetur adipscing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 14,
                "fontWeight": "500",
                "lineHeight": 21,
              },
            ]
          }
          type="button"
        >
          BUTTON TEXT
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 12,
                "fontWeight": "400",
                "lineHeight": 18,
              },
            ]
          }
          type="caption"
        >
          Caption text
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 12,
                "fontWeight": "500",
                "lineHeight": 18,
              },
            ]
          }
          type="overline"
        >
          OVERLINE TEXT
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 20,
                "fontWeight": "500",
                "lineHeight": 30,
              },
            ]
          }
          type="h6"
        >
          VARIATIONS
        </Text>
      </View>
      <View
        margin={24}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 24,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#F4AB34"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#F4AB34",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
            ]
          }
          type="body1"
        >
          body1. Set diffente color
        </Text>
      </View>
      <View
        margin={0}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 0,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
            ]
          }
          type="body1"
        >
          body1. Withou marginBottom spacingNone
        </Text>
      </View>
      <View
        margin={64}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 64,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
            ]
          }
          type="body1"
        >
          body1. Withou marginBottom spacingHuge
        </Text>
      </View>
      <View
        margin={4}
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": 4,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 0,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 0,
            },
          ]
        }
      >
        <Text
          color="#333333"
          style={
            Array [
              Object {
                "backgroundColor": "#FFF",
                "color": "#333333",
                "fontSize": 16,
                "fontWeight": "400",
                "lineHeight": 24,
              },
            ]
          }
          type="body1"
        >
          body1. Withou marginBottom spacingMicro
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
>>>>>>> origin/update/react
`;
